README.md

```diff
@@ -172,6 +172,21 @@ The default is 1 member to 1 group (deterministic run).
 :x: No implementation planned <br/>
 :soon: Work in progress
 
+# HofX case as a verification tool
+0. HofX case can be run separately as an independant verification tool.
+
+1. Instructions on how to run this case are as follows:
```

Comment:
Add the instructions for the hofx3d.yaml under configuration (line 138) and describe what options the user can use: Which observations and setting the path.